src/frontends/tensorflow_common/src/op/matrix_inverse.cpp

```diff
@@ -0,0 +1,34 @@
+// Contributed by Hongbo Wei
+// https://github.com/hongbo-wei
+// Good first issue: https://github.com/openvinotoolkit/openvino/issues/22957
+
+#include "openvino/op/inverse.hpp"
+
+#include "common_op_table.hpp"
+
+using namespace std;
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+OutputVector translate_matrix_inverse_op(const NodeContext& node) {
+    auto op_type = node.get_op_type();
+    TENSORFLOW_OP_VALIDATION(node, op_type == "MatrixInverse", "Internal error: incorrect usage of translate_matrix_inverse_op.");
+    auto equation = node.get_attribute<std::string>("equation"); 
```

Comment:
MatrixInverse has no such attributes.
Please see documetation and check what attributions need to be extracted: https://www.tensorflow.org/api_docs/python/tf/raw_ops/MatrixInverse

Reply:
Looked through the documentation, cannot find any attributions. I assume you mean by parameter?
I try to extract the "input" attribute